dspace-spring-rest/src/test/java/org/dspace/app/rest/CommunityRestRepositoryIT.java

```diff
@@ -273,11 +278,40 @@ public void findAllSubCommunities() throws Exception{
                 .andExpect(jsonPath("$._embedded.communities", Matchers.not(Matchers.containsInAnyOrder(
```

Comment:
1. Please add a comment here that you are checking that the following communities are not present.
2. You should not use `containsInAnyOrder` to check something is not present. You should use
```
not(anyOf(
   hasItem(...),
   hasItem(...),
   hasItem(...) 
))
```